### DISCUSSION TITLE
discord

### DISCUSSION THREAD

**Discussion by fairlawn206** [supabase/supabase#14189] [Questions]
sorry my questions are so stupid, I am very new. I wanted to hop into the discord to see what is going on and ask some questions but I am stuck in a loop. I really don't know anything about discord. Anyway the app launches and it's the supbase discord and there's a message that says ' claim this before you can speak on this channel' or something like that. So I press the button claim, and then I get a message, 'this is already c;laimed', so I go back to the discord app and it says 'claim this before you can speak on this channel.' Second verse same as the first. What am I doing wrong?

**Comment by ghost** [comment]
Try leaving and rejoining the Discord server. https://discord.gg/yNZn7WJr I would also recommend force closing Discord beforehand just in case. 

How to force close Discord:
1. Locate the Discord icon in the system tray, usually found in the bottom-right corner of the screen, near the clock.
2. Right-click on the Discord icon to open the context menu.
3. From the context menu, select "Quit Discord".

Then just reopen and rejoin.
